variables.tf

```diff
@@ -308,3 +308,225 @@ variable "check_ec2_imdsv2" {
   type        = bool
   default     = false
 }
+
+variable "check_access_keys_rotated" {
+  description = "Enable access-keys-rotated rule"
+  type        = bool
+  default     = true
+}
+
+variable "access_key_max_age" {
+  description = "Maximum number of days without rotation."
+  type        = number
+  default     = 90
+}
+
+variable "check_cmk_backing_key_rotated" {
+  description = "Enable cmk_backing_key_rotation_enabled rule"
+  type        = bool
+  default     = true
+}
+
+variable "check_backup_plan_min_frequency_and_min_retention" {
+  description = "Enable backup_plan_min_frequency_and_min_retention_check rule"
```

Comment:
What is backup plan min frequency and min retention?

Reply:
This is for [backup-plan-min-frequency-and-min-retention-check](https://docs.aws.amazon.com/config/latest/developerguide/backup-plan-min-frequency-and-min-retention-check.html) While we don't use backup plans, this is I think the first rule I added before thinning down the list a bit. Instead of removing it, I left it in since the work was completed for it.

Reply:
Are other teams using backup plans? There is _a lot_ in this PR. I'm inclined not to include things that we don't use in.